VIRTUAL_VACATION/vacation.pl

```diff
@@ -695,6 +722,12 @@ sub check_and_clean_from_address {
     exit(0);
 }
 $logger->debug("Email headers have to: '$to' and From: '$from'");
+
+if ($to =~ /^.*($novacation_pattern).*/i) { 
```

Comment:
With the empty default value for `$novacation_pattern`, this will match everything, prevent vacation mails from ever being sent.